tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -5,118 +5,164 @@ summary: Learn about the log auditing feature for the TiDB Cloud console.
+| InviteUserToProject | Invite a user to join the project |
+| DeleteInvitationToProject | Delete a user's invitation to join the project |
+| ResendInvitationToProject | Resend the invitation to join the project for a user |
+| ConfirmJoinProject | The invited user confirms joining the project |
+| DeleteUserFromProject | Delete a joined user from the project |
+| CreateProject | Create a project |
+| CreateProjectCIDR | Create a new project CIDR |
+| CreateAWSVPCPeering | Create an AWS VPC Peering |
+| DeleteAWSVPCPeering | Delete an AWS VPC Peering |
+| CreateGCPVPCPeering | Create a GCP VPC Peering |
+| DeleteGCPVPCPeering | Delete a GCP VPC Peering |
+| CreateAWSPrivateEndPoint | Create AWS private endpoint |
+| EditAWSPrivateEndPoint | Edit AWS private endpoint |
+| DeleteAWSPrivateEndPoint | Delete AWS private endpoint |
+| SubscribeAlerts | Subscribe alerts |
+| UnsubscribeAlerts | Unsubscribe alerts |
+| CreateDatadogIntegration | Create datadog integration |
+| DeleteDatadogIntegration | Delete datadog integration |
+| CreateVercelIntegration | Create vercel integration |
+| DeleteVercelIntegration | Delete vercel integration |
+| CreatePrometheusIntegration | Create prometheus integration |
+| DeletePrometheusIntegration | Delete prometheus integration |
+| CreateCluster | Create a cluster |
+| DeleteCluster | Delete a cluster |
+| PauseCluster | Pause a cluster |
+| ResumeCluster | Resume a cluster |
+| ScaleCluster | Scale a cluster |
+| DownloadTiDBClusterCA | Download TiDB cluster CA certificate |
+| OpenWebSQLConsole | Connect to TiDB Cluster through Web SQL |
+| SetRootPassword | Set the root password of TiDB cluster |
+| UpdateIPAccessList | Update the IP access list of TiDB cluster |
+| DeleteAccessList | Delete the IP access list of TiDB cluster |
+| SetAutoBackup | Set the automatic backup mechanism of TiDB cluster |
+| DoManualBackup | Do manual backup of TiDB cluster |
+| DeleteBackupTask | Delete backup task |
+| DeleteBackup | delete backup file |
+| RestoreFromBackup | Restore to TiDB cluster based on the backup file |
+| RestoreFromTrash | Restore to TiDB cluster based on the backup files in the trash |
+| ImportDataFromAWS | Import data from AWS |
+| ImportDataFromGCP | Import data from GCP |
+| CreateMigrationJob | Create migration job |
+| SuspendMigrationJob | Suspend migration job |
+| ResumeMigrationJob | Resume migration job |
+| DeleteMigrationJob | Delete migration job |
+| ShowDiagnose | Show diagnose information |
+| DBAuditLogAction | Set the activity of database audit log |
+| AddDBAuditFilter | Add database audit log filter |
+| DeleteDBAuditFilter | Delete database audit log filter |
+| EditProject | Edit a project's information |
+| DeleteProject | Delete a project |
+| BindSupportPlan | Bind support plan |
+| CancelSupportPlan | Cancel support plan |
+| UpdateOrganizationName | Update organization name |
+| CreatePrivateEndpointService | Create private endpoint service |
```

Comment:
move CreatePrivateEndpointService and DeletePrivateEndpointService before CreateAWSPrivateEndPoint